backend/src/models/performances.py

```diff
@@ -0,0 +1,15 @@
+from typing import TYPE_CHECKING
+from sqlalchemy import String
+from sqlalchemy.orm import Mapped, mapped_column
+
+from backend.src.models import BaseModel, TimestampAbstractModel
+
+if TYPE_CHECKING:
+    from backend.src.models.theatrical_roles import TheatricalRole
+
+
+class Performance(BaseModel, TimestampAbstractModel):
```

Comment:
When I upload in a good quality the model should be modified. Will add picture in a few hours

Reply:
Added the picture